### DISCUSSION TITLE
Make the Menubar / Landing Page customisable

### DISCUSSION THREAD

**Discussion by DerPicknicker** [music-assistant/support#1670] [Feature requests and ideas]
Hey, 

First of all this MA is crazy. All in all I really like the modern look. It is easy to setup and makes music streaming really comfortable.

For example: 
![image](https://github.com/music-assistant/support/assets/64746593/e02b84b5-3862-455b-8239-822d77022537)

Here I would really like to seen an option to disable entries and re-order them. From my point of view. I only need albums, Search, Radio, Playlists, and settings. 

The same is for the landing page, I would disable all entries except, albums, last played, radio and playlists. And here it would be also great if i can re-order them. I usally use radio more often than my own music. 

I think my point is clear. Please comeback if you need more information.

**Comment by OzGav** [comment]
The UI is under development although right now we lack a frontend dev so the update rate will be slow. In time this will be considered. Thanks for the suggestion.

**Reply by DerPicknicker** [reply]
Totally understandable. I already love the modern ui but this was so obvious for me that I had to post this as an suggestion

**Reply by OzGav** [reply]
Just following this up for other readers. The menu bar is customisable in the settings, you can show the options you wish. The order is set however.

The landing page is not yet configurable

**Reply by OzGav** [reply]
The home page is now configurable in 2.5.0 or later

**Comment by coffeemanuksr** [comment]
Also the ability to add custom link/icon to the menu.

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
